tests/integration_tests/security/demo_seccomp/Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Amazon Firecracker team <firecracker-devel@amazon.com>"]
 edition = "2018"
 
 [dependencies]
-libc = ">=0.2.39"
+libc = "0.2.66"
```

Comment:
Can t we use the same as in prod? (i.e 0.2.121)?

Reply:
See https://github.com/firecracker-microvm/firecracker/pull/3248#discussion_r1035015892